modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/repo/DetectorMappingRepository.java

```diff
@@ -44,4 +45,6 @@
     void disableDetectorMapping(String id);
 
     void deleteDetectorMapping(String id);
+
+    void deleteMappingByDetectorUUID(UUID uuid);
```

Comment:
we can rename this deleteMappingsByDetectorUUID since multiple mappings can be attached to the same UUID.